src/client/js/components/Admin/SlackIntegration/CustomBotWithoutProxySettingsAccordion.jsx

```diff
@@ -195,7 +195,7 @@ const CustomBotWithoutProxySettingsAccordion = ({
               <label className="mb-1"><p className="border-info slack-connection-log-title pl-2">Logs</p></label>
               <textarea
                 className="form-control card border-info slack-connection-log-body rounded-lg"
-                value={value}
+                defaultValue={value}
```

Comment:
FIX
```
Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
```

Reply:
あれ、これしたら test button 押した時に log が流れないのでは。
もしそうであるなら
```suggestion
<textarea
                value={value}
                readOnly
/>
```
が正しい改修だと思います

Reply:
失礼しました。修正しました。